python/ray/autoscaler/_private/kubernetes/config.py

```diff
@@ -160,6 +177,18 @@ def _parse_resource(resource):
         return int(resource_str)
 
 
+def _parse_memory_resource(resource):
+    resource_str = str(resource)
+    try:
+        return int(resource_str)
+    except ValueError:
+        pass
+    memory_size = re.sub(r"([KMGTP]+)", r" \1", resource_str)
+    number, unit_index = [item.strip() for item in memory_size.split()]
+    unit_index = unit_index[0:1]
```

Comment:
Why `unit_index[0:1]` as opposed to `unit_index[0]`? 
(nit)

Reply:
Updated.